meadow-client/src/Simulation.purs

```diff
@@ -190,7 +86,7 @@ simulationPane state =
       ]
     )
   where
-  transErrors = transactionErrors state.marloweState.transaction.validity
+  transErrors = transactionErrors (view (_marloweState <<< _Head <<< _transaction <<< _validity) state)
```

Comment:
Does this function ever look at anything other than the current head state? Because if not, you can (and should!) just pass it in at the top of the function.

Reply:
with some neatening I just put them inline